<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStartGame</class>
 <widget class="QDialog" name="DialogStartGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EzPuzzles - Start a new game.</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <widget class="FrameImageSelector" name="frameImageSelector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="FormSettingsLikeFifteen" name="formSettingsLikeFifteen">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Like 15</string>
      </attribute>
     </widget>
     <widget class="FormSettingsMineSweeper" name="formSettingsMineSweeper">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Mine Sweeper</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameImageSelector</class>
   <extends>QFrame</extends>
   <header>widgets/FrameImageSelector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormSettingsMineSweeper</class>
   <extends>QWidget</extends>
   <header>widgets/FormSettingsMineSweeper.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormSettingsLikeFifteen</class>
   <extends>QWidget</extends>
   <header>widgets/FormSettingsLikeFifteen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogStartGame</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogStartGame</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
